modules/nw-ovn-k-anp-troubeshooting.adoc

```diff
@@ -0,0 +1,36 @@
+//module included in the following assemblies:
+//
+//networking/network_security/AdminNetworkPolicy/nw-ovn-k-anp-troubleshooting.adoc
+:_mod-docs-content-type: REFRENCE
+[id="anp-troubleshooting_{context}"]
+= Checking creation of ANP
+
+To check that your `AdminNetworkPolicy` (ANP) and `BaselineAdminNetworkPolicy` (BANP) are created correctly, check the status outputs of the following commands: `oc describe anp` or `oc describe banp`.
+
+A good status will indicate that `OVN DB plumbing was successful` and that `SetupSucceeded`
+
+.Example ANP with a good status
+[source,terminal]
+----
+...
+Conditions:
+Last Transition Time:  2024-06-08T20:29:00Z
+Message:               Setting up OVN DB plumbing was successful
+Reason:                SetupSucceeded
+Status:                True
+Type:                  Ready-In-Zone-ovn-control-plane Last Transition Time:  2024-06-08T20:29:00Z
+Message:               Setting up OVN DB plumbing was successful
+Reason:                SetupSucceeded
+Status:                True
+Type:                  Ready-In-Zone-ovn-worker
+Last Transition Time:  2024-06-08T20:29:00Z
+Message:               Setting up OVN DB plumbing was successful
+Reason:                SetupSucceeded
+Status:                True
+Type:                  Ready-In-Zone-ovn-worker2
+...
+----
+
+If the plumbing is unsuccessful, you will see an error reported from the respective zone controller on the status.
```

Comment:
I dont think you need "you will see"
```suggestion
If plumbing is unsuccessful, an error is reported from the respective zone controller on the status.
```

Reply:
Also, follow up question is the status just saying there was an error? If so I think you could leave that part out. So,
```suggestion
If plumbing is unsuccessful, an error is reported from the respective zone controller.
```